app/home/component/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { CARD_PREVIEW_IMAGE_SIZE_WIDTH, CARD_PREVIEW_IMAGE_SIZE_HEIGHT } from "../const";
+
+import { CardType } from "./Card/Card";
+
+import { PopularMovie } from "@/app/lib/http/movie/movie.service";
+
+export const convertMovieResponseToCard = (movies: PopularMovie[] | undefined): CardType[] => {
```

Comment:
It seems to be quite specific to Card component rather than util function to be consumed globally or bigger scope. I'd put this under `Card` component where it belongs to

Reply:
Renamed Card, CardList, etc... to "MovieCard" related names.

Reply:
Ah kk that makes sense. You might as well leave a comment that explains the context under on the function